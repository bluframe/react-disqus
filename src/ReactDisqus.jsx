// @flow

import React from "react"

const style = {
  marginTop: "20px"
}

const ReactDisqus = () => (
  <div>
    <div
      id="disqus_thread"
      style={style}
    />

    <noscript>
      {`Please enable JavaScript to view the `}
      <a
        href="https://disqus.com/?ref_noscript"
        rel="nofollow"
      >
        {`comments powered by Disqus.`}
      </a>
    </noscript>
  </div>
)

export default ReactDisqus
