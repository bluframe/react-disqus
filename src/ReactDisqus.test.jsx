import React from "react"
import ReactDisqus from "./ReactDisqus"
import { shallow } from "enzyme"

describe("ReactDisqus", () => {
  it("renders", () => {
    const wrapper = shallow(<ReactDisqus />)

    expect(wrapper).toMatchSnapshot()
  })
})
