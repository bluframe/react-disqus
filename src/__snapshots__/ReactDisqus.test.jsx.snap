// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ReactDisqus renders 1`] = `
ShallowWrapper {
  "complexSelector": ComplexSelector {
    "buildPredicate": [Function],
    "childrenOfNode": [Function],
    "findWhereUnwrapped": [Function],
  },
  "length": 1,
  "node": <div>
    <div
      id="disqus_thread"
      style={
        Object {
          "marginTop": "20px",
        }
      }
    />
    <noscript>
      Please enable JavaScript to view the 
      <a
        href="https://disqus.com/?ref_noscript"
        rel="nofollow"
      >
        comments powered by Disqus.
      </a>
    </noscript>
  </div>,
  "nodes": Array [
    <div>
      <div
        id="disqus_thread"
        style={
          Object {
            "marginTop": "20px",
          }
        }
      />
      <noscript>
        Please enable JavaScript to view the 
        <a
          href="https://disqus.com/?ref_noscript"
          rel="nofollow"
        >
          comments powered by Disqus.
        </a>
      </noscript>
    </div>,
  ],
  "options": Object {},
  "renderer": ReactShallowRenderer {
    "_context": Object {},
    "_element": <ReactDisqus />,
    "_forcedUpdate": false,
    "_instance": null,
    "_newState": null,
    "_rendered": <div>
      <div
        id="disqus_thread"
        style={
          Object {
            "marginTop": "20px",
          }
        }
      />
      <noscript>
        Please enable JavaScript to view the 
        <a
          href="https://disqus.com/?ref_noscript"
          rel="nofollow"
        >
          comments powered by Disqus.
        </a>
      </noscript>
    </div>,
    "_rendering": false,
    "_updater": Updater {
      "_renderer": [Circular],
    },
    "getRenderOutput": [Function],
    "render": [Function],
  },
  "root": [Circular],
  "unrendered": <ReactDisqus />,
}
`;
